apps/src/applab/applab.js

```diff
@@ -46,7 +46,11 @@ var TestResults = studioApp.TestResults;
  */
 var Applab = module.exports;
 
-
+//console history (probably not the best way to store?)
+Applab.dc = {
+  'hist': [],
+  'i': 0
+};
```

Comment:
Let's use more verbose names here; this code gets minified before it's deployed, so there's no need to spare characters.  How about something like this?

```javascript
Applab.debugConsoleHistory = {
  history: [],
  currentHistoryIndex: 0
};
```